### ISSUE TITLE
feat: support cursor within Review mode

### ISSUE THREAD

**Issue by jooyoungseo** [xability/maidr-legacy#511]
Currently, you cannot use LeftArrow or RightArrow keys within Review mode. Screen reader users need to navigate character by character when confirming values. Please support cursor within the Review mode text field.
